app/controlplane/api/workflowcontract/v1/crafting_schema.proto

```diff
@@ -103,3 +105,32 @@ message Annotation {
   // This value can be set in the contract or provided during the attestation
   string value = 2;
 }
+
+// A policy to be applied to this contract
+message PolicyAttachment {
+  // policy reference. It must consist of an identifier or URI reference
+  string ref = 1 [(buf.validate.field).string.min_len = 1];
```

Comment:
what do you see as first ref, a local path ref? if so we might want to validate that's file://

Reply:
I was thinking just the policy `name`, since they need to exist previously.

Reply:
Actually, this could evolve to a `oneof` with `name`, `path`, and `embedded` options.

Reply:
Adding the oneof already to be prepared.